File: discord-music-bot/src/main/java/net/masterzach32/discord_music_bot/api/RandomCat.java
package net.masterzach32.discord_music_bot.api;

import com.mashape.unirest.http.HttpResponse;
import com.mashape.unirest.http.JsonNode;
import com.mashape.unirest.http.Unirest;
import com.mashape.unirest.http.exceptions.UnirestException;

public class RandomCat {
	
	private String url;

	public RandomCat() {
		try {
			HttpResponse<JsonNode> json = Unirest.get("http://random.cat/meow").asJson();
			url = json.getBody().getArray().getJSONObject(0).getString("file");
		} catch (UnirestException e) {
			e.printStackTrace();
		}
	}
	
	public String getUrl() {
		return url;
	}
}

File: discord-music-bot/src/main/java/net/masterzach32/swagbot/api/RandomCat.kt
package net.masterzach32.swagbot.api

import com.mashape.unirest.http.Unirest
import com.mashape.unirest.http.exceptions.UnirestException

class RandomCat {

    var url: String? = null
        private set

    init {
        try {
            val json = Unirest.get("http://random.cat/meow").asJson()
            url = json.body.array.getJSONObject(0).getString("file")
        } catch (e: UnirestException) {
            e.printStackTrace()
        }

    }
}